### PR TITLE
[Configs] Fix collect routes from Symfony router on AddRouteAnnotationRector

### PR DESCRIPTION
**Pull request by samsonasik**
Fixes https://github.com/rectorphp/rector/issues/8385
Fixes https://github.com/rectorphp/rector-symfony/issues/554

### CODE REVIEW

File: rules/Configs/Rector/ClassMethod/AddRouteAnnotationRector.php [hunk 1]
```diff
@@ -84,6 +86,7 @@ public function refactor(Node $node): ?Node
             }
 
             $hasChanged = true;
+            $this->docBlockUpdater->updateRefactoredNodeWithPhpDocInfo($classMethod);
```
Comment (samsonasik) [inline rules/Configs/Rector/ClassMethod/AddRouteAnnotationRector.php:89]: @TomasVotruba this is the reason the docblock was not updated, this patch fix it.

**Comment by samsonasik** [comment]
Fixed 🎉 /cc @fpetrovic , @TomasVotruba the php token notice seems a php-doc-parser bug, unrelated to this:

```
1 test triggered 4 PHP warnings:

1) /home/runner/work/rector-symfony/rector-symfony/vendor/phpstan/phpdoc-parser/src/Parser/TokenIterator.php:132
Undefined array key 40

Triggered by:

* Rector\Symfony\Tests\Configs\Rector\ClassMethod\AddRouteAnnotationRector\AddRouteAnnotationRectorTest::test#4
  /home/runner/work/rector-symfony/rector-symfony/rules-tests/Configs/Rector/ClassMethod/AddRouteAnnotationRector/AddRouteAnnotationRectorTest.php:14
```


re https://github.com/rectorphp/rector-symfony/actions/runs/7613166670/job/20732426711?pr=577#step:5:27

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I am merging it ;)

File: rules/Configs/Rector/ClassMethod/AddRouteAnnotationRector.php (hunk 1 above)
Comment (TomasVotruba) [reply rules/Configs/Rector/ClassMethod/AddRouteAnnotationRector.php:94]: Thank you 🙏

**Pull request merged**